cpg_workflows/stages/large_cohort.py

```diff
@@ -489,6 +490,195 @@ def queue_jobs(self, cohort: Cohort, inputs: StageInput) -> StageOutput | None:
         return self.make_outputs(cohort, data=self.expected_outputs(cohort), jobs=[j])
 
 
+@stage(required_stages=[Combiner])
+class ShardVds(CohortStage):
+    def expected_outputs(self, cohort: Cohort) -> dict[str, Path]:
+        if sharded_vds_version := config_retrieve(['large_cohort', 'output_versions', 'sharded_vds'], default=None):
+            sharded_vds_version = slugify(sharded_vds_version)
+
+        sharded_vds_version = sharded_vds_version or get_workflow().output_version
+        return {
+            f'{contig}': cohort.analysis_dataset.prefix()
+            / get_workflow().name
+            / 'sharded_vds'
+            / sharded_vds_version
+            / f'{contig}.vds'
+            for contig in [f'chr{i}' for i in range(1, 23)] + ['chrX', 'chrY', 'chrM']
+        }
+
+    def queue_jobs(self, cohort: Cohort, inputs: StageInput) -> StageOutput | None:
+        from cpg_workflows.large_cohort import generate_coverage_table
+
+        j = get_batch().new_job(
+            'ShardVds',
+            (self.get_job_attrs() or {}) | {'tool': HAIL_QUERY},
+        )
+        j.image(image_path('cpg_workflows'))
```

Comment:
best not to use `image_path('cpg_workflows')`, it's rarely the right image. Use `config_retrieve(['workflow', 'driver_image'])`